starlite/connection.py

```diff
@@ -74,3 +75,32 @@ def auth(self) -> Auth:
     @property
     def query_params(self) -> Dict[str, Any]:  # type: ignore[override]
         return parse_query_params(self)
+
+    async def receive_json(self, mode: str = "text") -> Any:
+        """
+        Exact copy of the `starlette` method, but using `orjson.loads()`.
+        """
+        if mode not in {"text", "binary"}:
+            raise RuntimeError('The "mode" argument should be "text" or "binary".')
```

Comment:
Should probably be one of our exceptions or a new exception of ours that inherits from both StarliteException and RuntimeError.

Reply:
For sure, will incorporate when I next iterate on this.

Reply:
Went with `InternalServerException`